Source/CesiumRuntime/Public/Cesium3DTileset.h

```diff
@@ -31,20 +45,32 @@ class CESIUMRUNTIME_API ACesium3DTileset : public AActor,
   ACesium3DTileset();
   virtual ~ACesium3DTileset();
 
+  /**
+   * The type of source from which to load this tileset.
+   */
+  UPROPERTY(EditAnywhere, Category = "Cesium", meta = (DisplayName = "Source"))
+  ETilesetSource TilesetSource = ETilesetSource::FromCesiumIon;
```

Comment:
Adding and honoring this property will break non-ion tilesets. They'll stop working after upgrading to the new version of Cesium for Unreal because this property defaults to `FromCesiumIon`.

I think we can fix this by implementing backward compatibility for our property serialization. Some docs here:
https://docs.unrealengine.com/en-US/ProgrammingAndScripting/ProgrammingWithCPP/UnrealArchitecture/VersioningAssetsAndPackages/index.html

I'll take a crack.

Reply:
Hey that turned out to be pretty easy, done in 427ef46.